ingeniamotion/communication.py

```diff
@@ -884,9 +891,23 @@ def load_firmware_canopen(
             status_callback = partial(self.logger.info, "Load firmware status: %s")
         if progress_callback is None:
             progress_callback = partial(self.logger.info, "Load firmware progress: %s")
-        net.load_firmware(
-            int(drive.target), fw_file, status_callback, progress_callback, error_enabled_callback
-        )
+        if fw_file.endswith(self.ENSEMBLE_FIRMWARE_EXTENSION):
+            self.__load_ensemble_fw_canopen(
```

Comment:
Do you test it in a real drive? I think it will not work because the info scan disconnects the drives but you need drives connected to load the FW...

Reply:
No, I did not test it in real drives yet. But, the info scan should not disconnect the drives.

Reply:
I think this must be tested before merging. Until we have the real file test will not be 100% real, but it must not change a lot.